crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -790,7 +794,7 @@ impl TypedTransaction {
     /// Returns true if the transaction was impersonated (using the impersonate Signature)
     #[cfg(feature = "impersonated-tx")]
     pub fn is_impersonated(&self) -> bool {
-        self.signature() == IMPERSONATED_SIGNATURE
+        to_alloy_signature(self.signature()) == IMPERSONATED_SIGNATURE
```

Comment:
make `self.signature()` return an alloy signature imo, any reason why we cannot port the typed transaction fields to be alloy?

Reply:
thought about doing this, but migrating `typedtransaction` so early would make us migrate everything in one go—ideally we want to avoid this